Snippets/ASQ/ASQ_7/Configuration/MessageWrapperSerializer.cs

```diff
@@ -0,0 +1,36 @@
+﻿namespace ASQ_7.Configuration
+{
+    using NServiceBus;
+
+    public class MessageWrapperSerializer
+    {
+        #region SerializerAndMessageWrapperSerializer
+        public void ConfigureEndpointA()
+        {
+            var endpointA = new EndpointConfiguration("Endpoint-A");
+            // serialize the messages using the XML serializer:
+            endpointA.UseSerialization<XmlSerializer>();
+            // enables handling messages from Endpoint-B
+            endpointA.AddDeserializer<JsonSerializer>();
+
+            endpointA.UseTransport<AzureStorageQueueTransport>()
+                .ConnectionStringName("ASQ-ConnectionString")
+                // wrap messages in JSON to maintain compatibility with existing messages
+                .SerializeMessageWrapperWith<JsonSerializer>();
+        }
+
+        public void ConfigureEndpointB()
```

Comment:
What's the purpose in trying to highlight both functions in the same snippet? What are we trying to convey to the user?